### ISSUE TITLE
Types extending primitive types are not considered primitive

### ISSUE THREAD

**Issue by GregSilverman** [dkpro/dkpro-cassis#93]
**Describe the bug**

When selecting view, the error `invalid literal for int() with base 10: 'CARDINAL'` is thrown.

**To Reproduce**

Run following:

```
fname = '20132.txt.xmi'
typesystem = 'TypeSystem.xmi'
with open(fname, 'rb') as f:
    cas = load_cas_from_xmi(f, typesystem=typesystem)
 
view = cas.get_view('Analysis')
print([x for x in view.select("biomedicus.v2.UmlsConcept")])
```

[20132.txt.xmi.zip](https://github.com/dkpro/dkpro-cassis/files/3872491/20132.txt.xmi.zip)
[TypeSystem.xml.zip](https://github.com/dkpro/dkpro-cassis/files/3872498/TypeSystem.xml.zip)

**Expected behavior**
Expect to see annotations in view (NB: this had worked in previous versions)

**Error message**
See above.

**Please complete the following information:**
 - Version: v0.2.3

 - OS: MacOS: High Sierra

**Comment by jcklie** [comment]
The problem is that `edu.umn.biomedicus.numbers.NumberType` is not seen as a primitive type even though it  extends `uima.cas.String` which . We need to check inheritance in `is_primitive`.

**Comment by jcklie** [comment]
I merged it in master and will maybe releae it this or early next week.

**Comment by reckart** [comment]
AFAIK strings are the only primitive type which can be extended - and the idea in doing so is that you can define a set of fixed values which the string feature can assume.

**Comment by GregSilverman** [comment]
Hi, looks like the fix in 0.2.4 worked. Thanks!
